index.yaml

```diff
@@ -23,11 +23,12 @@ entries:
     description: A Jaeger Helm chart for Kubernetes
     digest: 948a87dd5cb4ac4095a5b0e2c817ccbb193709b49668104993b5642398c7e27b
     home: https://jaegertracing.io
-    icon: https://camo.githubusercontent.com/afa87494e0753b4b1f5719a2f35aa5263859dffb/687474703a2f2f6a61656765722e72656164746865646f63732e696f2f656e2f6c61746573742f696d616765732f6a61656765722d766563746f722e737667
+    icon: https://artifacthub.io/image/bffa8705-c91a-48d5-92fc-8054d9478d77@2x
```

Comment:
I don't know the origin of that icon on the hub. It's better to refer to official logo here https://github.com/cncf/artwork/tree/main/projects/jaeger

e.g. https://raw.githubusercontent.com/cncf/artwork/main/projects/jaeger/icon/color/jaeger-icon-color.png

Reply:
Should be good now.